DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/Controllers/RepositoryController.cs

```diff
@@ -207,5 +209,28 @@ private void AddClaimsToCookie(Claim[] claims)
                 GetRequestContext.SetCookie(claim.Type, claim.Value);
             }
         }
+
+        List<CollaboratorWithUserIdModel> AddUserIdToCollaboratorIfExists(IEnumerable<CollaboratorModel> gitHubCollaboratorsExceptUser)
+        {
+            var collaborators = new List<CollaboratorWithUserIdModel>();
+
+            var users = _securityContextQuery.GetAllUsers();
```

Comment:
Moved query to context to action method and passed it as an argument. Commit added